ddprof-lib/src/main/cpp/vmStructs_dd.cpp

```diff
@@ -98,6 +99,16 @@ namespace ddprof {
     }
   }
 
+  void VMStructs_::initCriticalJNINatives() {
+#ifdef __aarch64__
+    // aarch64 does not support CriticalJNINatives
+    JVMFlag* flag = JVMFlag::find("CriticalJNINatives", {JVMFlag::Type::Bool});
```

Comment:
IIUC, `-XX:+CriticalJNINatives` requires also `-XX:+UnlockExperimentalVMOptions` :/

This would be a hard sell to our customers.